test/topology/suite.yaml

```diff
@@ -4,3 +4,4 @@ cluster_size: 3
 extra_scylla_config_options:
     authenticator: AllowAllAuthenticator
     authorizer: AllowAllAuthorizer
+default_markers: not medium and not large
```

Comment:
When do we plan to run these tests? We should run them regularly, if not during gating. Maybe the daily build.

Reply:
Talk with the releng team to help with such a thing